### ISSUE TITLE
String interpolation doesn't work at the end of string

### ISSUE THREAD

**Issue by MadcapJake** [gabordemooij/citrine#85]
```
catcall := 'meow'.
Pen write: 'The cat goes $$catcall'.
```
Crashes with this bug:
```
Parse error, unexpected ( ( t/020-get.ctr: 20 )
Expected ).
```
Actually, sometimes it seems to segfault instead of printing that error.

Not sure if this is related but the line number doesn't match too.

**Comment by gabordemooij** [comment]
Fixed and added unit test.

**Comment by gabordemooij** [comment]
Thanks for reporting this issue.

**Comment by gabordemooij** [comment]
has been released in 0.7.2
